### PR TITLE
mkdir command with spec file

### PR DESCRIPTION
**Pull request by ekeast**
Here is my code for the mkdir.rb file and the spec file. This is my first contribution to an open source project, so please bear with me if there are any mistakes!

### CODE REVIEW

File: lib/packages/mkdir.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    # iterates through the directorys in the current directory to make sure that the desired directory does not exist
+    ls.each do |x|
+      if x == new_directory
+        directory_present = true
+        break
+      end
+    end
+
+    # creates the directory if it does not exist already
+    # NOTE: I thought about showing an error message if the directory already exists, but the original touch command does not do this.
```
Comment (faraazahmad) [inline lib/packages/mkdir.rb]: *mkdir

Also,
```
$ mkdir test
$ mkdir test
mkdir: cannot create directory ‘test’: File exists
```
😄

**Comment by faraazahmad** [comment]
Awesome! You might wanna check your comments a little
;)

**Comment by faraazahmad** [comment]
using `unless...else...end` is not recommended. Use an `if...else...end` block instead

**Comment by ekeast** [comment]
Thanks a lot! I've made those changes and committed. Please feel free to let me know if further changes are required.

**Pull request merged**